src/main/java/com/woowacamp/soolsool/core/cart/controller/CartItemController.java

```diff
@@ -0,0 +1,33 @@
+package com.woowacamp.soolsool.core.cart.controller;
+
+import com.woowacamp.soolsool.core.cart.code.CartItemResultCode;
+import com.woowacamp.soolsool.core.cart.dto.request.CartItemSaveRequest;
+import com.woowacamp.soolsool.core.cart.service.CartItemService;
+import com.woowacamp.soolsool.global.auth.dto.LoginUser;
+import com.woowacamp.soolsool.global.common.ApiResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/cart-items")
+public class CartItemController {
+
+    private final CartItemService cartItemService;
+
+    @PostMapping
+    public ResponseEntity<ApiResponse<Void>> saveCartItem(
+        @LoginUser final Long memberId,
+        @RequestBody final CartItemSaveRequest cartItemSaveRequest
+    ) {
+        cartItemService.saveCartItem(memberId, cartItemSaveRequest);
+
+        return ResponseEntity.status(HttpStatus.CREATED)
+            .body(ApiResponse.of(CartItemResultCode.CART_ITEM_ADD_SUCCESS, null));
+    }
```

Comment:
```suggestion
        return ResponseEntity.status(HttpStatus.OK)
            .body(ApiResponse.from(CartItemResultCode.CART_ITEM_ADD_SUCCESS));
    }
```

CREATED는 Location 헤더를 넣어주는게 국룰이더라구요. 조회 API가 없어서 못 넣는다면 200으로~

>   The action performed by the POST method might not result in a  resource that can be identified by a URI. In this case, either 200 (OK) or 204 (No Content) is the appropriate response status, depending on whether or not the response includes an entity that describes the result.


전 키보드로 null을 치는 순간 폭발하는 프로그램을 만들고 싶어요.

Reply:
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ 오 조회 API가 없을 경우 200으로 반환하는 컨벤션을 잊고 있었네요!!
그리고 Response의 from도 존재를 잊고 있었습니다!!
감사합니다!!
(null 폭발 프로그램은 좋은 것 같네여 ㅋㅋㅋㅋㅋㅋㅋㅋ)